packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md

```diff
@@ -1,17 +1,85 @@
-## How to enable SSO in TeamsFx Bot projects
+# Enable single sign-on for bot applications
```

Comment:
Shall we mention that this doc is for latest version?

Reply:
You mean this doc is for the latest toolkit version?

Reply:
Yes, we may mention this doc is for extension >= xxx

Reply:
This readme is scaffolded by us, probably not necessary to me.